docs/usage/configuration-options.md

```diff
@@ -1644,6 +1644,21 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
```

Comment:
I think if we keep both `languages` and `categories` support at the same time then we need to document the differences/overlaps

Reply:
We should also clarify if these packageRules are only applied *after* extract. e.g. people might try to configure something like `fileMatch`